CodeEdit/Features/CEWorkspace/Views/AddCETaskView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AddCETaskView.swift
+//  CodeEdit
+//
+//  Created by Axel Martinez on 3/4/24.
+//
+
+import SwiftUI
+import Collections
+
+struct AddCETaskView: View {
+    @Environment(\.dismiss)
+    var dismiss
+
+    @Binding private var settings: CEWorkspaceSettingsData.TasksSettings
+
+    @State private var task: CETask
+
+    init(workingDirectory: String, settings: Binding<CEWorkspaceSettingsData.TasksSettings>) {
+        self._settings = settings
+        self._task = State(initialValue: CETask(
+            target: "My Mac",
+            workingDirectory: workingDirectory
+        ))
+    }
+
+    var body: some View {
+        VStack(spacing: 0) {
+            CETaskFormView(
+                task: $task
+            )
+            Spacer()
+            Divider()
+            HStack {
+                Button("Remove...") {
```

Comment:
```suggestion
                Button("Remove") {
```

Reply:
This is the design we agreed on, I think we should comment on this before making this change

Reply:
Austin just explained that to me:
> You see it in buttons, but you also see it in menubar items. An ellipsis (…) will represent that action will not be taken whenever the user clicks the item, but rather will open a window or sheet to perform the action. For example. File > Print… this has an ellipsis because a sheet will open first whereas Save does not because the action happens immediately after selection.

I wasn't aware of that.